server/models.py

```diff
@@ -828,6 +828,17 @@ class BatchResult(BaseModel):
     __table_args__ = (PrimaryKeyConstraint("batch_id", "contest_choice_id"),)
 
 
+# Records when the jurisdiction finalizes their batch results for a round.
+class BatchResultsFinalized(BaseModel):
+    jurisdiction_id = Column(
+        String(200), ForeignKey("jurisdiction.id", ondelete="cascade"), nullable=False,
+    )
+    round_id = Column(
+        String(200), ForeignKey("round.id", ondelete="cascade"), nullable=False
+    )
+    __table_args__ = (PrimaryKeyConstraint("jurisdiction_id", "round_id"),)
```

Comment:
I wonder if a slightly different design here might be slightly more future-proof and maybe even useful now. For discussion, not saying it's necessary, consider:
- a `BatchResultsInfo` table
- a `finalized_at` field
- any other possibly useful metadata fields?

Reply:
Hmm, I can't think of any other useful metadata off the top of my head. Would you create the `BatchResultsInfo` at a different time, or still upon finalizing?

I'm not _too_ worried about future-proofing here because we can always migrate.

Reply:
I raised it in case you might think of other metadata to attach, and specifically that you might create the row earlier... If neither makes sense now, I'm very comfortable punting on this altogether.